src/ui/hexedit.cc

```diff
@@ -52,18 +52,20 @@ void HexEdit::recalculateValues() {
   if (dataBytesCount_ + startOffset_ >= 0x100000000LL) {
     addressBytes_ = 8;
   }
-  addressWidth_ = addressBytes_ * 2 * charWidht_ + verticalAreaSpaceWidth_;
+  addressWidth_ = addressBytes_ * 2 * charWidth_ + verticalAreaSpaceWidth_;
 
   // plus one for only addr
   rowsCount_ = (dataBytesCount_ + bytesPerRow_ - 1) / bytesPerRow_ + 1;
   rowsOnScreen_ =
       (viewport()->height() - horizontalAreaSpaceWidth_ * 2) / charHeight_;
 
-  spaceAfterByte_ = charWidht_ / 2;
-  hexAreaWidth_ = bytesPerRow_ * byteCharsCount_ * charWidht_ +
+  // Spacing needs to be odd for proper border alignment.
+  spaceAfterByte_ = (charWidth_ / 3 | 1);
```

Comment:
Hmm, I don't agree with this, because the rule is simple - all bit operators are below arithmetic operators. We should assume that the reader knows C++ enough and if not he can just google this.  Especially with division, which has a very high priority.